deployment/fedora-package-x64/Dockerfile

```diff
@@ -8,22 +8,28 @@ ARG SDK_VERSION=2.2
 ENV SOURCE_DIR=/jellyfin
 ENV ARTIFACT_DIR=/dist
 
-# Prepare Fedora build environment
-RUN dnf update -y \
- && dnf install -y @buildsys-build rpmdevtools dnf-plugins-core libcurl-devel fontconfig-devel freetype-devel openssl-devel glibc-devel libicu-devel nodejs \
- && dnf copr enable -y @dotnet-sig/dotnet \
+# Prepare Fedora environment
+RUN dnf update -y
+
+# Install build dependencies
+RUN dnf install -y @buildsys-build rpmdevtools dnf-plugins-core libcurl-devel fontconfig-devel freetype-devel openssl-devel glibc-devel libicu-devel nodejs wget git
```

Comment:
again, why separate `RUN`?

Reply:
Same as above, avoids repeating the update (this one was close to 20 mins) each time a dep was updated.